app/enterprise/changelog.md

```diff
@@ -5,6 +5,127 @@ no_version: true
 skip_read_time: true
 ---
 
+## 2.1.3.0
+**Release Date** 2020/08/25
+
+Kong Enterprise 2.1.3.0 version includes 2.1.0.0 (beta) features, fixes, known issues, and workarounds. See the [2.1.0.0 (beta)](/enterprise/changelog/#2100-beta/) changelog for more details.
+
+### Features
+
+**Note: Feature updates for Kong Enterprise 2.1.3.0 version includes [2.1.0.0 (beta)](/enterprise/changelog/#features) features.**
+
+#### Kong Gateway
+* Inherited changes from OSS Kong in releases 2.0.x, 2.1.0, 2.1.1, 2.1.2, and 2.1.3.
+* Workspaces code has been refactored for performance. The feature should work the same for most users.
+* TLS version may be specified when using tls to connect to a Postgres db.
+
+#### Kong Manager
+* Open ID Connect (`openid-connect`) can now be used with [Mapping Service Directory Groups to Kong Roles](/enterprise/latest/kong-manager/service-directory-mapping/) using `config.authenticated_groups_claim`.
+* Can now see the `created_at` timestamp for consumer credentials.
+* Provide a warning when a user's session is about to expire.
+* Generate an RBAC token for an administrator rather than letting them set one.
+* Plugin forms now have docs links to the Plugins Hub.
```

Comment:
Does it make sense to have a link to `/hub` for the Plugins Hub?

Reply:
yes. done.